scanner/laravel.go

```diff
@@ -104,3 +114,54 @@ func extractPhpVersion() (string, error) {
 
 	return "", fmt.Errorf("could not find php version")
 }
+
+func extractConnections(path string) (DatabaseKind, bool, bool) {
+	/*
+		Determine the default db
+		Determine whether redis connection is desired
+			returns ( db, redis, skipDB )
+	*/
+
+	// Get File Content
+	file, err := os.Open(path)
+	if err != nil {
+		return 0, false, true
+	}
+	defer file.Close() //skipcq: GO-S2307
+
+	// Set up Regex to match
+	// -not commented out, with DB_CONNECTION
+	dbReg := regexp.MustCompile("^ *DB_CONNECTION *= *[a-zA-Z]+")
+	// -not commented out with redis keyword
+	redisReg := regexp.MustCompile("^[^#]*redis")
```

Comment:
static regular patterns (and its regex.MustCompile() calls) are ideal as global variables as they can catch regex syntax errors earlier. More details at https://stackoverflow.com/a/72185182